app/src/components/pages/ProfessorsList.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { DataGrid, GridColDef } from '@mui/x-data-grid';
+import { parseCalendarTeacher } from 'utils/utils';
+import classData from 'data/clean.json';
+import { Box } from '@mui/material';
+import SearchIcon from '@mui/icons-material/Search';
+import { styled, alpha } from '@mui/material/styles';
+import InputBase from '@mui/material/InputBase';
+import { useState } from 'react';
+import { ICalendarTeacherItem } from 'components/shared/interfaces/timetable.interfaces';
+import Link from '@mui/material/Link';
+
+const columns: GridColDef[] = [
+  { field: 'id', headerName: 'ID', width: 70 },
+  { field: 'teacherName', headerName: 'Username', width: 130 },
+  { field: 'courseId', headerName: 'Course', width: 130 },

```

Comment:
I would drop courseID and term since we (eventually) won't be using courses to define teachers. Just show the username and Id. There will be duplicates for now but that's okay, they won't be present in the actual implementation